client/src/index.js

```diff
@@ -2,20 +2,21 @@ import React from 'react'
 import { render } from 'react-dom'
 import { Provider } from 'react-redux'
 import { ConnectedRouter } from 'react-router-redux'
-import { theme, MuiThemeProvider } from 'material-ui/styles'
 
 import store, { history } from './base/store'
 import App from './containers/app'
 
 // import material-ui theme
+import MuiThemeProvider from 'material-ui/styles/MuiThemeProvider'
+import './constants/styles/styles.js'
```

Comment:
Is this being used here?

Reply:
I thought that it was, but removing it apparently changes nothing. It's been removed since it does nothing.